bmcs/src/test/resources/reference.conf

```diff
@@ -0,0 +1,32 @@
+akka {
+  http {
+    client {
+      parsing.max-content-length = 4294967296 #4GB
+      parsing.illegal-header-warnings = off
+    }
+  }
+  stream {
+    materializer {
+      subscription-timeout {
+        mode = warn
+        timeout = 10s
+      }
+    }
```

Comment:
You cannot override defaults in a `reference.conf`, because then if they are applied or not are dependent on class path ordering, put them in an `application.conf` instead.

Reply:
fixed and pushed.